aws-lambda-java-events/pom.xml

```diff
@@ -69,6 +71,18 @@
             <version>2.36.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.skyscreamer</groupId>
+            <artifactId>jsonassert</artifactId>
+            <version>1.5.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-lambda-java-serialization</artifactId>
```

Comment:
Can you move the test to `aws-lambda-java-tests` instead?

Reply:
Done